test/endpoints/automations.test.js

```diff
@@ -1,15 +1,35 @@
 import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+import axios from 'axios';
 import chai, { expect } from 'chai';
+import * as resource from '../../server/modules/allocations';
+import automationData from '../mocks/automationData';
 import app from '../../server';
 
 chai.use(chaiHttp);
 
 describe('Tests for automation endpoints\n', () => {
-  it('Should return automation data with a 200 response code', (done) => {
+  before(async () => {
+    sinon.stub(axios, 'get').callsFake(
+      () => new Promise((resolve) => {
+        resolve(automationData);
+      }),
+    );
+    sinon.stub(Date, 'now').callsFake(() => 1542376484108);
+    const result = await resource.fetchNewPlacements('Placed - Awaiting Onboarding', 3);
+    console.log('====================================');
+    console.log(result);
+    console.log('====================================');
+  });
+
+  it('Should return automation data with a 200 response code', async (done) => {
```

Comment:
@janetnim  don't use async and done together use either one of them
```suggestion
  it('Should return automation data with a 200 response code', (done) => {
```
this will eliminate the error you where getting in circle ci
```bash
 Should return automation data with a 200 response code:
     Error: Resolution method is overspecified. Specify a callback *or* return a Promise; not both.
      at done (test/endpoints/automations.test.js:40:9)
      at Test.Request.callback (node_modules/superagent/lib/node/index.js:716:12)
      at parser (node_modules/superagent/lib/node/index.js:916:18)
      at IncomingMessage.res.on (node_modules/superagent/lib/node/parsers/json.js:19:7)
      at endReadableNT (_stream_readable.js:1081:12)
      at process._tickCallback (internal/process/next_tick.js:63:
```

Reply:
I've made the said changes on using async and done at the same time